### DISCUSSION TITLE
[Feature] customize the traceid

### DISCUSSION THREAD

**Discussion by wuzhihui1123** [apache/skywalking#8232] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Is it possible to provide a way to customize the traceid? When I create the TraceContext, I provide the traceid by myself

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
There is not an out-of-box way you could set this up
